Installation issue: gdal

### Steps to reproduce the issue

@adamjstewart

```console
spack install gdal@2.4.4 os=centos7 %gcc@13
```


### Error message

<details><summary>Error message</summary>

<pre>
/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1649:5-11: info: tuple ignored:
  #sup@2

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1649:5-11: info: tuple ignored:
  #sup@2

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1649:5-11: info: tuple ignored:
  #sup@2

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1649:5-11: info: tuple ignored:
  #sup@2

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/spack/solver/concretize.lp:1613:5-11: info: tuple ignored:
  #sup@205

[+] /usr (external glibc-2.34-bvg6gnpldjcedowsfhr7yr6qoldmboej)
[+] /usr (external glibc-2.34-dtaqd4e3lpequ6yh2mkvp6e7qxjmc372)
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/gcc-runtime-13.2.0-p3ytakljc7w37llw6hdzi2nyqqgpuldl
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/gcc-runtime-13.2.0-sej6atsxbrcgihuzhnyvueabnrez6jyu
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/geos-3.12.1-a2irdgo35tleyd3kqtwc2o5apohvnhm2
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/pkgconf-2.2.0-dzpsfnxvrhqtg22uxpj2tb6jvhc34tqj
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libbsd-0.10.0-t57k5bnnq2qgkdmeo2sclu7vx2yvj3nu
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/gmake-4.4.1-4qcmieobg5n6idcvrdqvasgauubsln3c
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/nghttp2-1.57.0-wyzrf437ck7ftfd4g3lmldk4cf5xyg2b
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/expat-2.6.2-rvkjvlhenseop65lbpluj77relf5tlgx
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/proj-5.2.0-y5uwabnvpe23ykkafag466op2n7xinu4
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/json-c-0.16-wwhwtze5fivvx3ebxnafrkq53ic4mdsj
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/ncurses-6.5-dp4kc7itqrvxbb2k3bpnd6tvly4mu64h
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/openssl-3.3.0-bfjppgwcb4i3azwzy76etfaguleujkew
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/readline-8.2-2zrwzdwrbra4sdvnn4njiwls6se622gg
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/curl-8.7.1-sctuvg2lgcgbxwtbuhpyrpqokl6dugqb
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w
[+] /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/sqlite-3.43.2-oy7k5kno4yv3rnszfq3mf7o2otf6pbda
==> Installing gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy [23/23]
==> No binary for gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy found: installing from source
==> Using cached archive: /mnt/apps/aristotle/spack-0.22.0/spack/var/spack/cache/_source-cache/archive/a3/a383bd3cf555d6e1169666b01b5b3025b2722ed39e834f1b65090f604405dcd8.tar.xz
==> Ran patch() for gdal
==> gdal: Executing phase: 'autoreconf'
==> gdal: Executing phase: 'configure'
==> gdal: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

12 errors found in build log:
     228     checking for _SC_PHYS_PAGES... yes
     229     checking for deflateInit_ in -lz... yes
     230     checking for inflateCopy in -lz... yes
     231     using libz library from /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu
     232     checking for ld used by GCC... /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/ld -m elf_x86_64
     233     checking if the linker (/mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/ld -m elf_x86_64) is GNU ld... yes
  >> 234     checking for shared library run path origin... /bin/sh: ./config.rpath: No such file or directory
     235     done
     236     checking for iconv... yes
     237     checking for working iconv... yes
     238     checking for iconv declaration...
     239              extern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
     240     using ICONV_CPP_CONST=""

     ...

     6204    make[3]: Entering directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/wfs'
     6205    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../.. -I../gml  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/sp
             ack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo
             -3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux
             -centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/ap
             ps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n
             4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrwfsdriver.lo ogrwfsdriver.cpp
     6206    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6207    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../.. -I../gml  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/sp
             ack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo
             -3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux
             -centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/ap
             ps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n
             4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrwfsdatasource.lo ogrwfsdatasource.c
             pp
     6208    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6209    r2000.cpp: In member function 'virtual int DWGFileR2000::CreateFileMap()':
  >> 6210    r2000.cpp:811:27: error: 'numeric_limits' is not a member of 'std'
     6211      811 |                      std::numeric_limits<long>::max() - tmpOffset.first > previousObjHandleOffset.first) ||
     6212          |                           ^~~~~~~~~~~~~~
  >> 6213    r2000.cpp:811:42: error: expected primary-expression before 'long'
     6214      811 |                      std::numeric_limits<long>::max() - tmpOffset.first > previousObjHandleOffset.first) ||
     6215          |                                          ^~~~
  >> 6216    r2000.cpp:811:42: error: expected ')' before 'long'
     6217      811 |                      std::numeric_limits<long>::max() - tmpOffset.first > previousObjHandleOffset.first) ||
     6218          |                                          ^~~~
     6219          |                                          )
     6220    r2000.cpp:810:21: note: to match this '('
     6221      810 |                 if( (tmpOffset.first >= 0 &&
     6222          |                     ^
  >> 6223    r2000.cpp:816:18: error: expected ')' before 'if'
     6224      816 |                 }
     6225          |                  ^
     6226          |                  )
     6227      817 |                 if( (tmpOffset.second >= 0 &&
     6228          |                 ~~
     6229    r2000.cpp:810:19: note: to match this '('
     6230      810 |                 if( (tmpOffset.first >= 0 &&
     6231          |                   ^
  >> 6232    r2000.cpp:824:13: error: expected primary-expression before '}' token
     6233      824 |             }
     6234          |             ^
     6235    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../.. -I../gml  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/sp
             ack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo
             -3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux
             -centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/ap
             ps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n
             4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrwfslayer.lo ogrwfslayer.cpp
     6236    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../.. -I../gml  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/sp
             ack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo
             -3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux
             -centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/ap
             ps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n
             4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrwfsfilter.lo ogrwfsfilter.cpp
     6237    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6238    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I. -I.. -I../.. -Ilibopencad -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehc
             bogy/spack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpe
             g-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/sit
             e/linux-centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I
             /mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucj
             wbwpe5n4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrcaddriver.lo ogrcaddriver.cp
             p
     6239    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
  >> 6240    make[5]: *** [GNUmakefile:15: ../../../o/r2000.lo] Error 1
     6241    make[5]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/cad/libopencad/dwg'
  >> 6242    make[4]: *** [../../../../GDALmake.opt:683: dwg-target] Error 2
     6243    make[4]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/cad/libopencad'
  >> 6244    make[3]: *** [../../../GDALmake.opt:683: libopencad-target] Error 2
     6245    make[3]: *** Waiting for unfinished jobs....
     6246    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../.. -I../gml  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/sp
             ack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo
             -3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux
             -centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/ap
             ps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n
             4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrwfsjoinlayer.lo ogrwfsjoinlayer.cpp
     6247    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6248    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/jml'
     6249    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I..  -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/sqlite-3.43.2-oy7k5kno4yv3rnszfq3mf7o2otf6p
             bda/include  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opua
             qczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc
             -13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/arist
             otle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/in
             clude -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucj
             wbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -DSQLITE_HAS_COLUMN_METADATA -c -o ../o/ogrsqlitedriver.lo ogrsqlitedriver.cpp
     6250    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++

     ...

     6263    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6264    make -C gft
     6265    make[3]: Entering directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/gft'
     6266    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../..  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/p
             ort -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r
             4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux-centos7-
             x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/apps/aristo
             tle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aig
             c6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrgftdriver.lo ogrgftdriver.cpp
     6267    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6268    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/cad'
  >> 6269    make[2]: *** [../../GDALmake.opt:683: cad-target] Error 2
     6270    make[2]: *** Waiting for unfinished jobs....
     6271    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../..  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/p
             ort -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r
             4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux-centos7-
             x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/apps/aristo
             tle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aig
             c6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrgftdatasource.lo ogrgftdatasource.cpp
     6272    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6273    /bin/sh /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/libtool --mode=compile --silent --tag=CXX /mnt/apps/aristotle/spack-0.22.0/spack/lib/spack/env/gcc/g++ -I/tmp/pa
             tta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/port -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gcore -I/tmp/patta/spack-stage-gdal-2.4.4-r
             6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/alg -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy
             /spack-src/ogr/ogrsf_frmts -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/gnm -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/apps -DH
             AVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -g -O2  -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogica
             l-op -Wshadow -Werror=vla -Wdate-time -Wnull-dereference -Wduplicated-cond -Wextra-semi -Wfloat-conversion -Wmissing-prototypes -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-o
             perator-names -Wzero-as-null-pointer-constant -Wsuggest-override -Wimplicit-fallthrough  -I.. -I../..  -DGNM_ENABLED -I/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/p
             ort -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r4opuaqczf4qubs3fnjxwqkfxynkdgg -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libjpeg-turbo-3.0.0-4r
             4opuaqczf4qubs3fnjxwqkfxynkdgg/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libgeotiff-1.4.3-t53qq2apvbwalhxiwkogbqolqcmi3s4w/include -I/mnt/apps/aristotle/site/linux-centos7-
             x86_64/gcc-13.2.0/libtiff-4.5.1-mpegju7bwgc4i33phyagltfbh54agmux/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl -I/mnt/apps/aristo
             tle/site/linux-centos7-x86_64/gcc-13.2.0/libpng-1.6.39-kmfu3ea5u235lmtjug235liwtr4yzarl/include -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aig
             c6uu -I/mnt/apps/aristotle/site/linux-centos7-x86_64/gcc-13.2.0/zlib-ng-2.1.6-5cp3e7t77ucjwbwpe5n4v75p4aigc6uu/include  -DGDAL_COMPILATION -c -o ../o/ogrgftlayer.lo ogrgftlayer.cpp
     6274    cc1plus: warning: command-line option '-Wmissing-prototypes' is valid for C/ObjC but not for C++
     6275    make[2]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/frmts/pdf'

     ...

     6317          | ^
     6318    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/mvt'
     6319    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/wfs'
     6320    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/vfk'
     6321    make[3]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts/sqlite'
     6322    make[2]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr/ogrsf_frmts'
  >> 6323    make[1]: *** [GNUmakefile:58: sublibs] Error 2
     6324    make[1]: Leaving directory '/tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-src/ogr'
  >> 6325    make: *** [GNUmakefile:89: ogr-target] Error 2

See build log for details:
  /tmp/patta/spack-stage-gdal-2.4.4-r6ke547mnd76oka5rfd2tdtiuehcbogy/spack-build-out.txt
</pre></details>


### Information on your system

* **Spack:** 0.22.1
* **Python:** 3.9.18
* **Platform:** linux-rocky9-zen2
* **Concretizer:** clingo
* **Configuration files:** [spack.zip](https://github.com/user-attachments/files/16560196/spack.zip)


### Additional information

[spack-build-out.txt](https://github.com/user-attachments/files/16560221/spack-build-out.txt)
[spack-build-env.txt](https://github.com/user-attachments/files/16560236/spack-build-env.txt)


### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/gdal/package.py
+++ b/var/spack/repos/builtin/packages/gdal/package.py
@@ -100,8 +100,8 @@ class Gdal(CMakePackage, AutotoolsPackage, PythonExtension):
         version("2.0.1", sha256="2564c91ed8ed36274ee31002a25798f5babc4221e879cb5013867733d80f9920")
         version("2.0.0", sha256="91704fafeea2349c5e268dc1e2d03921b3aae64b05ee01d59fdfc1a6b0ffc061")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
 
     # Optional dependencies
     variant("archive", default=False, when="@3.7:", description="Optional for vsi7z VFS driver")
@@ -430,16 +430,19 @@ class Gdal(CMakePackage, AutotoolsPackage, PythonExtension):
     depends_on("googletest@1.10:", type="test")
 
     # https://gdal.org/development/rfc/rfc98_build_requirements_gdal_3_9.html
-    msg = "GDAL requires C++17 support"
-    conflicts("%gcc@:7", msg=msg)
-    conflicts("%clang@:4", msg=msg)
-    conflicts("%msvc@:19.14", msg=msg)
+    with default_args(when="@3.9:", msg="GDAL requires C++17 support"):
+        conflicts("%gcc@:7")
+        conflicts("%clang@:4")
+        conflicts("%msvc@:19.14")
 
     # https://gdal.org/development/rfc/rfc68_cplusplus11.html
-    msg = "GDAL requires C++11 support"
-    conflicts("%gcc@:4.8.0", msg=msg)
-    conflicts("%clang@:3.2", msg=msg)
-    conflicts("%msvc@:13", msg=msg)
+    with default_args(when="@2.3:", msg="GDAL requires C++11 support"):
+        conflicts("%gcc@:4.8.0")
+        conflicts("%clang@:3.2")
+        conflicts("%msvc@:13")
+
+    # https://github.com/OSGeo/gdal/issues/8693
+    conflicts("%gcc@11:", when="@:3.6")
 
     # https://github.com/OSGeo/gdal/issues/5994
     conflicts("~png", when="@3:3.5.0")

